code/game/objects/items/charter.dm

```diff
@@ -60,7 +60,7 @@
 	user << "Your name has been sent to your employers for approval."
 	// Autoapproves after a certain time
 	response_timer_id = addtimer(src, "rename_station", approval_time, \
```

Comment:
can you just make it one line, i mean... there's a line right below that's at least double the size, and reading procs like this is a pain